AUCTIONS_WINNERS.md

```diff
@@ -0,0 +1,135 @@
+      ]
+    }
+  ]
+}
+```
+
+Important keys here are `finishEnd` which is the last block at which the auction will take place, `auctionIndex` which will act as your point of reference for that specific auction, and `leasePeriods` which are the available lease period indexes that may be bid on for the specific `auctionIndex`. 
+
+** Details of each key **
+
+`finishEnd`: This is the last block at which the auction will take place, and also the most important piece of data needed. This will allow you to query the block at which the auction has ended and will give you all the events that have taken place during that block. 
+
+`auctionIndex`: Also known as the `AuctionCounter`, the index will be a pointer or ID to a specific auction. 
+
+`leasePeriods`: These are the available lease period indexes that may be bid on for the specific `auctionIndex`. You can use this to create two seperate relationships explained below. 
+
+
+## Relationships
+
+With the information you are now tracking there are two main relationships to note. 
+
+1. `auctionIndex` => `leasePeriods` (Lease Periods that are actively being auctioned off)
+
+2. `paraId` => `leasePeriods` (Lease Periods that have been won).
+
+
+## Walkthrough
+
+### Relay-chain Dev Setup for auctions (SKIP if you already have a development setup)!
+-------------
+
+The begining of this guide will start by briefly introducing setting up a simple parachain enviornment if you dont already know how or dont have one set for local developement. 
+
+Start by cloning [polkadot](https://github.com/paritytech/polkadot) and checking out the `rococo-v1` branch. NOTE: Before compiling make sure to adjust the [EndingPeriod](https://github.com/paritytech/polkadot/blob/rococo-v1/runtime/rococo/src/lib.rs#L745) to `100`, and [LeasePeriod](https://github.com/paritytech/polkadot/blob/rococo-v1/runtime/rococo/src/lib.rs#L761) to `100` so the auction time is fit for local development. You can then follow this [gist](https://gist.github.com/emostov/a58f887fce6af8a9b4aa2421114836c5) to get your alice and bob dev validator nodes up. Then you can call those extrinsic calls with the polkadot-js UI. 
+
+### Using Sidecar to find the auction winners
+
+`/experimental/paras/auctions/current` ENDPOINT
+
+An auction will either be in two phases `starting` or `ending`. During this period when querying the endpoint you will receive a `finishEnd` which will denote the last block where the `AuctionClosed` event will take place as well as the `Leased` event. These events will be under the `on_initialize` key. 
+
+It is important to index this block for the current `auctionIndex` because this will be your source of truth of where the winners of the auction are stored. 
+
+The next important key is the `leasePeriods` that corresponds to the current `auctionIndex`. We will use these available `leasePeriods` to compare the winning results and see which `paraId`'s took which slots. 
+
+By this point we have the below relationship. 
```

Comment:
Confused by the term relationship here. I think it is alluding to the fact that we can check the lease periods for a paraId and check the lease periods for an old auction to see if they overlap. Then if they overlap we know the paraId won the overlapping ones from the auction. 

However, now that I think about it, using the `Leased` events from the ending block is better is because there may be multiple auctions that cover overlapping lease periods (so testing for the overlap i mention could be misleading)

Given that, can we drop the `Relationships` section above and just recommend watching for events at the ending block?

Reply:
I definitely like this. It adds for a more open ended implementation while having instructions on what to watch for, and where.